client/signup/steps/design-picker/style.scss

```diff
@@ -57,4 +57,10 @@
 			}
 		}
 	}
+
+	body.is-section-signup.is-white-signup & {
```

Comment:
I think that this selector is actually unneccessary, `.action-buttons` should not have a background on this page on the old blue theme either. Since this is already wrapped in `.signup__step.is-design, .signup__step.is-design-setup-site` it will only affect the design step.

Reply:
A yes that's true, we don't want a white background ever.